src/views/start.html

```diff
@@ -14,6 +34,16 @@ <h1 class="govuk-heading-xl">GitHub Requests</h1>
                     <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
                 </svg>
             </a>
+            {{ govukInsetText({
+                text: "An email will be sent to the account authenticated to COLA (Cabinet Office Login Authentication app)."
```

Comment:
..."authenticated _with_ COLA"

and

COLA stands for Cabinet Office Login Application

Reply:
Ahh I meant just update the words in the brackets:

_An email will be sent to the account authenticated with COLA (Cabinet Office Login Application)._